apps/client/src/shared/api/domain/community/queries.ts

```diff
@@ -200,6 +202,20 @@ export const COMMUNITY_MUTATION_OPTIONS = {
         deleteCommentReply(postId, commentId, commentReplyId),
     });
   },
+
+  ADD_LIKE: (postId: string) => {
+    return mutationOptions<LikeAddResponse, Error, void>({
+      mutationKey: COMMUNITY_MUTATION_KEY.ADD_LIKE(postId),
+      mutationFn: () => postLike(postId),
+    });
+  },
```

Comment:
```suggestion
  ADD_LIKE: (postId: string) => {
    return mutationOptions({
      mutationKey: COMMUNITY_MUTATION_KEY.ADD_LIKE(postId),
      mutationFn: () => postLike(postId),
    });
  },
```
그러면 이 부분도 이렇게 사용해도 될것 같아요!